src/pages/sa-trx/[functionId]/forms/SaleTransactionForm.js

```diff
@@ -404,7 +404,8 @@ export default function SaleTransactionForm({ labels, access, recordId, function
         if (!newRow.itemId) return
         const itemPhysProp = await getItemPhysProp(newRow.itemId)
         const itemInfo = await getItem(newRow.itemId)
-        const ItemConvertPrice = await getItemConvertPrice(newRow.itemId)
+        const filteredMeasurements = measurements?.filter(item => item.msId === itemInfo?.msId)
```

Comment:
u are filtering based on msId not muId + u can take what u need from itemInfo no need to add filteredMeasurements